models/user_avatar.go

```diff
@@ -82,14 +82,19 @@ func (u *User) RealSizedAvatarLink(size int) string {
 		if u.Avatar == "" {
 			return DefaultAvatarLink()
 		}
+		if size > 0 {
+			return setting.AppSubURL + "/avatars/" + u.Avatar + "?size=" + strconv.Itoa(size)
```

Comment:
What does the `size` parameter do?

Reply:
nothing right now - but we should probably keep sending it for future options